pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -16,7 +16,9 @@
       hash ? "",
       pname,
       pnpmWorkspace ? "",
+      pnpmWorkspaces ? [ ],
       prePnpmInstall ? "",
+      pnpmInstallFlags ? [ ],
```

Comment:
Can you mention this with an example in the docs?

Reply:
Added a mention of this.